misc/write_fake_manifests.py

```diff
@@ -149,7 +184,7 @@ def write_master_ninja(master_ninja, targets):
       command='$cxx -MMD -MF $out.d $defines $includes $cflags -c $in -o $out',
       depfile='$out.d', deps='gcc')
     master_ninja.rule('alink', description='LIBTOOL-STATIC $out',
-      command='rm -f $out && libtool -static -o $out $in')
+      command='rm -f $out && ar rcs -o $out $in')
```

Comment:
I suppose this needs `libtools -static` on OS X and `ar rcs` on other POSIXen (check sys.platform)